benches/suites/raw_node.rs

```diff
@@ -1,13 +1,18 @@
-use criterion::{Bencher, Criterion};
-use raft::{storage::MemStorage, Config, RawNode};
+use criterion::{Bencher, Benchmark, Criterion, Throughput};
```

Comment:
I note that Criterion 0.3.0 (the new version) has no `Benchmark` struct, so maybe the previous way was more forwards compatible? https://docs.rs/criterion/0.3.0/criterion/index.html

Reply:
Done. `BenchmarkGroup` works perfectly on this :)